packages/nameguard-sdk/src/index.ts

```diff
@@ -601,6 +630,8 @@ export class NameGuard {
    *   1. The labelhash of the "childmost" label of a name.
    *   2. The complete parent name of the "childmost" label.
    *
+   * Returns `UninspectedNameGuardReport` if the resolved `name` is uninspected.
```

Comment:
@djstrong I think it will be nice to update this to more formally refer to the new `inspected` field rather than the abstract idea of being inspected / uninspected.